DOCS/man/input.rst

```diff
@@ -1754,8 +1754,8 @@ This list uses the event name field value, and the C API symbol in brackets:
     Start of playback after seek or after file was loaded.
 
 ``shutdown`` (``MPV_EVENT_SHUTDOWN``)
-    Sent when the player quits, and the script should terminate. Normally
-    handled automatically. See `Details on the script initialization and lifecycle`_.
+    Sent when the player quits or when a script terminates. Normally handled
```

Comment:
shutdown is sent to *all* clients when any script terminates? that's not what this means, is it?

Reply:
It's sent to the terminating script.

Reply:
That makes sense but isn't very clear from the text.

Reply:
> It's sent to the terminating script.

It's the other way around.

The shutdown event IS what terminates the script. A terminating script doesn't trigger a shutdown event.